src/frontend/components/blockSwitch/blockSwitch.tsx

```diff
@@ -0,0 +1,5 @@
+import React from "react";
+
+const BlockSwitch = () => <></>;
+
+export default BlockSwitch;
```

Comment:
Maybe parent components should just call Block and have no idea that it will be switched.